### ISSUE TITLE
Estimated delay in timetables

### ISSUE THREAD

**Issue by ue71603** [NeTEx-CEN/NeTEx#749]
In some timetables with already know that there will be a probable delay (e.g. due to construction). We would prefer not to change the arrival and departure time (Arrival/Time and Departure/Time). We were thinking about using CheckConstraints also for that.

However, it seems we don't  a CheckProcess or CheckService that reflects that.
```xml
<Call id="ch:1:Call:ch:1:ServiceJourney:33-15365-1-1_1_2" version="any" order="2">
    <ScheduledStopPointRef ref="ch:1:ScheduledStopPoint:8504416:3" version="any"/>
    <Arrival>
        <Time>17:30:00</Time>
        <ForAlighting>true</ForAlighting>
        <IsFlexible>false</IsFlexible>
        <CheckConstraint id="ch:1:CheckConstraint:2" version="any" order="1">
            <delays id="ch:1:delays:1">
                <CheckConstraintDelay id="ch:1:CheckConstraintDelay:1" version="any" order="1">
                    <alternativeTexts>
                        <AlternativeText attributeName="Name">
                            <Text lang="it">Ritardo stimato</Text>
                        </AlternativeText>
                        <AlternativeText attributeName="Name">
                            <Text lang="en">Estimated delay</Text>
                        </AlternativeText>
                        <AlternativeText attributeName="Name">
                            <Text lang="fr">Retard prévu</Text>
                        </AlternativeText>
                    </alternativeTexts>
                    <Name>Voraussichtliche Verspätung</Name>
                    <MinimumLikelyDelay>PT4M</MinimumLikelyDelay>
                </CheckConstraintDelay>
            </delays>
        </CheckConstraint>
    </Arrival>
```
We could use CheckProcess=none or other.

Questions:
1. Is this something to be stored in CheckConstraints
2. If yes, how to model the CheckProcess? New value?

**Comment by Aurige** [comment]
is it a delay for all ServiceJourneys of the calendar, or just on a specific (operational) day ?

**Comment by Aurige** [comment]
I recognise that it is not obvious
I would create a SJ1.1 for the dedicated period, and add a dedicated **Description** and maybe an assigned **Notice** (where you can mention and explain the delay) and of course updated passing times. 
SJ1.1 can also be linked to SJ1 using **derivedFromObjectRef**, and a version o SJ1 can have a **ServiceAlteration**=_replaced_ 

I guess that, even if in your example it is a delay, it is more a service reorganisation due to the construction/works and it may also be early, or cancelled stops, etc. or a mix of all

**Comment by ue71603** [comment]
<ServiceJourney id="444" version="1">
   <AvailabilityCondition id="allYear"/>
    < TrainNumber>444</TrainNumber>
</ServieJourney>

planned delay for July 

<ServiceJourney id="444-1" version="1">
   <AvailabilityCondition id="allYearexceptJuly"/>
    < TrainNumber>444</TrainNumber>
</ServieJourney>

<ServiceJourney id="444-2" version="1">
   <AvailabilityCondition id="July"/>
    < TrainNumber>444</TrainNumber>
   <ServiceAlteration>replaced</ServiceAlteration>
</ServieJourney>

<ServiceJourney id="444-2a" version="2">
   <AvailabilityCondition id="July"/>
    < TrainNumber>444</TrainNumber>
    <derivedFromObjectRef  ref="444-2" version="1"/>
</ServieJourney>

**Comment by Aurige** [comment]
yes, something like that ... I guess the the Description/Notice is also important for passenger information
Also I see a risk that a journey planner dos not managed the _<ServiceAlteration>replaced</ServiceAlteration>_ ... and use it in July. It may be nice to be explicit and set _isAvailable=false_ in _444-2_ **AvailabilityCondition**

**Comment by BredeD** [comment]
I think your use case are sinilar to what we use datedServiceJourney for, where we also can manage the links between multippel replacement buses to a previous planned train.

**Comment by ue71603** [comment]
see also: https://3.basecamp.com/3256016/buckets/2570434/messages/7575802156#__recording_7591006952

**Comment by ue71603** [comment]
DatedVehicleJourney can be used. One problem: These are month-long changes. So we would like to crate one for each operating day and we don't like to have a different mechanism than for ServiceJourney.
We then would do it with an AvailabilityCondition wit ValidDayBits.

![image](https://github.com/user-attachments/assets/33ef42d6-e06f-450d-acd5-5073bc2c1b85)


We should then perhaps update the definitions in NeTEx to clearly allow this. So we also don't really want to use the UicOperatingPeriod/ValidityDayBits.

**Comment by ue71603** [comment]
@trurlurl  Can you try to build a simple example (from an existing one with the derivedFromObject? As simple as possible.  Or add it to an existing example?
